test/integration/targets/ansible-doc/collections/ansible_collections/testns/testcol/roles/testrole/meta/main.yml

```diff
@@ -28,3 +28,6 @@ argument_specs:
                 description: altopt1 description
                 type: "int"
                 required: true
+        examples: |-
```

Comment:
wouldn't role argspec examples have to be YAML by def?

Reply:
The content should better be a string with YAML in it. If you put pure YAML here, neither formatting nor comments will survive the transformation process, since everything will be converted to JSON inbetween.